<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>research_book</class>
 <widget class="QDialog" name="research_book">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(&quot;:/images/bg_login2.jpg&quot;);
background-position: center</string>
  </property>
  <widget class="QPushButton" name="btn_voltar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>470</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgb(255, 82, 223);
font: 10pt &quot;Times New Roman&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Voltar</string>
   </property>
  </widget>
  <widget class="QLabel" name="texto_frase">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LayoutDirection::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0,79,0);
font: 9pt &quot;Times New Roman&quot;;
</string>
   </property>
   <property name="text">
    <string>&quot;Ler é sonhar pela mão de outrem.&quot;
             - Fernando Pessoa</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>399</width>
     <height>371</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="3">
     <widget class="QLabel" name="txt_inicio">
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 85, 0);
font: 14pt &quot;Times New Roman&quot;;</string>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="text">
       <string>Insira título, autor e formato do livro que você
deseja pesquisar.</string>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="txt_titulo">
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 85, 0);
font: 11pt &quot;Times New Roman&quot;;</string>
      </property>
      <property name="text">
       <string>Título</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QLineEdit" name="insert_titulo"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="txt_autor">
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 85, 0);
font: 11pt &quot;Times New Roman&quot;;</string>
      </property>
      <property name="text">
       <string>Autor</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="2">
     <widget class="QLineEdit" name="insert_autor"/>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QLabel" name="txt_formato">
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 85, 0);
font: 11pt &quot;Times New Roman&quot;;</string>
      </property>
      <property name="text">
       <string>Formato</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QComboBox" name="insert_formato">
      <property name="tabletTracking">
       <bool>false</bool>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::ContextMenuPolicy::DefaultContextMenu</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgb(255, 255, 255);</string>
      </property>
      <property name="inputMethodHints">
       <set>Qt::InputMethodHint::ImhNone</set>
      </property>
      <item>
       <property name="text">
        <string>Fisico</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Digital</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Audio</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="4" column="0" colspan="3">
     <widget class="QPushButton" name="btn_pesquisar">
      <property name="styleSheet">
       <string notr="true">background: rgb(0, 79, 0);
color: rgb(255, 255, 255);
font: 10pt &quot;Times New Roman&quot;;</string>
      </property>
      <property name="text">
       <string>Pesquisar</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="3">
     <widget class="QProgressBar" name="progresso_leitura">
      <property name="styleSheet">
       <string notr="true">background: rgb(0, 79, 0);
QProgressBar::chunk {
          background-color: #3498db;
        width: 20px;
};  </string>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="3">
     <widget class="QLabel" name="txt_resultado">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Times New Roman&quot;;
color: rgb(0, 79,0);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
